libcxx/test/libcxx/utilities/expected/expected.void/and_then.mandates.verify.cpp

```diff
@@ -55,6 +55,7 @@ void test() {
       // expected-error-re@*:* {{static assertion failed {{.*}}The result of f() must be a specialization of std::expected}}
```

Comment:
OK, so these tests are a mess. libc++ shouldn't be using verify tests like this. 

Given how brittle these tests are, and how little they actually meaningfully test today, I think it's fair to simply make them work again. However you can; let's just not modify the headers to work around these tests.